### PR TITLE
Fix for line.strip() when line is None

### PR DESCRIPTION
**Pull request by BrendanMartin**
A fix for issue [#576](https://github.com/tweepy/tweepy/issues/576) that was closed but never seemed to be fixed.

### CODE REVIEW

**Comment by byoung0589** [comment]
When is the next release?  Running 3.5.0 and getting this error, contemplating rolling back to 3.2.0 if the next release is far out.

File: tweepy/cursor.py [hunk 1]
```diff
@@ -121,13 +128,18 @@ def next(self):
...
-            if len(self.results) != 0:
+
+            if self.results != 0:
                 self.index += 1
-            self.results.append(result)
-            self.model_results.append(model)
+            self.results += 1
+            self.model_results += 1
         else:
+            print("DON'T KNOW WHAT TO DO!!!!")
```
Comment (lysol) [inline tweepy/cursor.py]: really?

Comment (BrendanMartin) [reply tweepy/cursor.py]: Sorry, this was a personal branch I accidentally committed to this PR lol

**Comment by Harmon758** [comment]
This should be fixed by https://github.com/tweepy/tweepy/commit/32fa592d4ae535baf6e1cf17658d207fb656b16b (#1027) in [3.7.0](https://github.com/tweepy/tweepy/releases/tag/3.7.0).

**Pull request closed without merging**